.travis.yml

```diff
@@ -4,18 +4,16 @@ python:
   - 2.7
 
 before_install:
-  - git submodule update --init --recursive
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
```

Comment:
Thanks @davehunt. I agree. I guess I'm trying to do two things here:

1. Allow the tests to pass now, as long as there are no pep8 issues.
2. Have a travis file that will work when we do decide to run browser tests again.

I suppose #2 doesn't really make sense, as we can always update this file later.

This means we also don't need `pip install -Ur requirements.txt` if all we are doing is the `flake8` test, correct?